compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/PlatformContext.skiko.kt

```diff
@@ -228,6 +237,27 @@ private object EmptyFocusManager : FocusManager {
     override fun moveFocus(focusDirection: FocusDirection) = false
 }
 
+private object UnimplementedDragAndDropManager : PlatformDragAndDropManager {
```

Comment:
Let's keep the pattern and move an object to `PlatformDragAndDropManager.Empty` (internal) and (optionally) TODOs as the default interface implementation

Reply:
Not sure I understand. There's no need for an "Empty" PlatformDragAndDropManager. The pattern is that "Empty" does nothing, but we want "Unimplemented" that throws. It should not be in  `PlatformDragAndDropManager.kt` because such an implementation is not the `PlatformDragAndDropManager`s concern.

Reply:
The idea behind "Empty" is to provide maximum basic/"empty" platform that doesn't crash, even if user call this API directly.

Otherwise, users can encounter crashes if they test a widget/screen that contains `dragAndDropSource`.

Regarding where it should be defined (companion `PlatformDragAndDropManager.Empty`, extension `PlatformDragAndDropManager.Empty`, function/class `EmptyPlatformDragAndDropManager`) - I don't have a preference.